scripts/scripts.js

```diff
@@ -318,7 +318,7 @@ export function getWindowSize() {
 export function addTopSpacingStyleToFirstMatchingSection(main) {
   const excludedClasses = ['static', 'spacer-container', 'feed-container', 'modal-fragment-container',
     'hero-banner-container', 'hero-career-container', 'breadcrumb-container', 'hero-horizontal-tabs-container',
-    'carousel-container', 'with-background-image', 'report-overview-container'];
+    'carousel-container', 'with-background-image', 'report-overview-container', 'header-no-top-padding'];
```

Comment:
Is `header-no-top-padding` a new variant? If yes,  how about just reusing `.section.newsroom` ?